openapi/payments_api.yaml

```diff
@@ -0,0 +1,535 @@
+openapi: 3.0.3
+info:
+  title: Payments API
+  description: >
+    <br/> MobilePay Payments API makes it easy for you to collect money from your
+    customers using MobilePay.
+
+    #### General notes
+
+    All dates use the (`ISO 8601-1:2019`) format.<br>
+    Date-time format - `YYYY-MM-DDTHH:mm:ssZ` or `YYYY-MM-DDTHH:mm:ss+00:00`.
+  contact:
+    email: developer@mobilepay.dk
+servers:
+  - url: https://api.mobilepay.dk
+    description: Production server (uses **live** data)
+  - url: https://api.sandbox.mobilepay.dk
+    description: Production server (uses **test** data)
+paths:
+  /v1/payments:
+    parameters:
+    - $ref: '#/components/parameters/correlationIdParam'
+    post:
+      summary: initiate new payment
+      operationId: initiate-payment
+      requestBody:
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/InitiatePaymentRequest'
+      responses:
+        '200':
+          description: OK
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/InitiatePaymentResponse'
+        '400':
+          description: Bad Request
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/ErrorResponse'
+        '401':
+          description: Unauthorized
+        '403':
+          description: Forbidden
+        '409':
+          description: Conflict
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/ErrorResponse'
+    get:
+      summary: fetch a list of payments
+      operationId: get-payments-list
+      description: >
+        Returns a list of all merchant's payments if no filters are provided in query parameters.
+        Can also return only payments of a specific payment point if `paymentPointId` is provided.
+      parameters:
+        - name: paymentPointId
+          in: query
+          description: Filter payments of a specific payment point
+          schema:
+            type: string
+            format: uuid
+        - name: pageNumber
+          in: query
+          schema:
+            type: integer
+            format: int32
+            minimum: 1
+            default: 1
+        - name: pageSize
+          in: query
+          description: Number of payments to be returned.
+          schema:
+            type: integer
+            format: int32
+            default: 100
+            minimum: 1
+            maximum: 1000
+        - name: stateFilter
+          in: query
+          description: Filter payments by state.
+          schema:
+            type: array
+            items:
+              enum:
+                - initiated
+                - reserved
+                - captured
+                - cancelledByMerchant
+                - captureStarted
+                - cancelledByUser
+                - cancelledBySystem
+                - terminated
```

Comment:
terminated and captureStarted were not present before

Reply:
removed